client/src/pages/ProfileSetupPage/ProfileSteps/ProfileStep3/DaysAvailableCheckBox/DaysAvailableCheckBox.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { Box, Checkbox,Typography } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import { orange } from '@material-ui/core/colors';
+
+const OrangeCheckbox = withStyles({
```

Comment:
since this is a functional component, maybe use `makeStyles`

https://material-ui.com/styles/api/#makestyles-styles-options-hook